web/src/lib/api-client/apiClient.test.ts

```diff
@@ -64,7 +61,7 @@ describe("apiClient", () => {
     await checkLicenseStatus(nameAndAddress);
     expect(mockAxios.post).toHaveBeenCalledWith(
       "/api/license-status",
-      { nameAndAddress, licenseTaskId: undefined },
```

Comment:
pebble - you should review the test and determine if they should be renamed, rewritten, or deleted.